presto-main/etc/config.properties

```diff
@@ -36,7 +36,8 @@ plugin.bundles=\
   ../presto-tpch/pom.xml, \
   ../presto-local-file/pom.xml, \
   ../presto-mysql/pom.xml,\
-  ../presto-postgresql/pom.xml
+  ../presto-postgresql/pom.xml,\
+  ../presto-cassandra/pom.xml
```

Comment:
Please remove this. We don't configure any Cassandra catalogs, so this just adds startup time needlessly.